extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -9,15 +10,14 @@ public function init() {
 		$this->backgroundImageHeight = null;
 		$this->wordmarkImageName = null;
 		$this->wordmarkImageUrl = null;
-		$this->errors = array();
+		$this->errors = [];
 
 		$this->dir = $this->wg->ContLang->getDir();
 		$this->mimetype = $this->wg->Mimetype;
 		$this->charset = $this->wg->OutputEncoding;
 	}
 
 	public function executeIndex() {
-		wfProfileIn( __METHOD__ );
 		global $wgLang, $wgOut;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)